src/components/Product.jsx

```diff
@@ -52,20 +101,35 @@ export const Product = (
       <div className="category">category / {category}</div>
       <h2>{name}</h2>
       <div className="product-image">
-        <img src={imageURL} alt="product" />
+        <img src={imageURL}/>
       </div>
       <div className="product-details">
-        <div className="product-price">${price}</div>
+        {inStock && (
+          <form>
+            <label>
+              <span>Quantity: </span>
+              <input
+                type="number"
+                value={form.quantity}
+                min="1"
```

Comment:
min = {1} not "1"

Reply:
Tried this with no change in results. When updating the form state, e.target.value reads a string.